### DISCUSSION TITLE
minimise demand charges

### DISCUSSION THREAD

**Discussion by Rujo** [amberelectric/public-api#210] [Q&A]
from July we will have demand charges. 
Batteries should allow us to drive these charges to zero.
However, it would be great if the API would show real-time data for consumption and export of electricity, to make sure that consumption during peak time is always zero.

**Comment by madpilot** [comment]
We can’t do realtime data.

If you are in Smartshift, the future Smartshift API will allow you to fetch data that is up to 1-minute old.

If you want realtime you need to use modbus or some other local protocol.
